examples/AST_L1T_00305032000040446_20150409135350_78838.json

```diff
@@ -0,0 +1,454 @@
+  "properties": {
+    "aster:processing_number": "78838",
+    "providers": [
+      {
+        "name": "NASA LP DAAC at the USGS EROS Center",
+        "roles": [
+          "producer",
+          "processor",
+          "licensor"
+        ],
+        "url": "https://doi.org/10.5067/ASTER/AST_L1T.003"
+      }
+    ],
+    "created": "2015-04-09T13:56:57.771000Z",
+    "platform": "terra",
+    "instruments": [
+      "aster"
+    ],
+    "eo:cloud_cover": 57.0,
+    "sat:orbit_state": "descending",
+    "view:sun_azimuth": 86.162211,
+    "view:sun_elevation": 75.830363,
+    "proj:epsg": 32648,
+    "aster:upper_left_quad_cloud_cover": 64.0,
+    "aster:upper_right_quad_cloud_cover": 44.0,
+    "aster:lower_left_quad_cloud_cover": 75.0,
+    "aster:lower_right_quad_cloud_cover": 44.0,
+    "view:off_nadir": 5.709,
+    "datetime": "2000-05-03T04:04:46.534000Z"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          102.685261260459,
+          15.7673228577021
+        ],
+        [
+          103.467912710542,
+          15.7742405668264
+        ],
+        [
+          103.472824966208,
+          15.1040095755409
+        ],
+        [
+          102.692678376984,
+          15.0973996420643
+        ],
+        [
+          102.685261260459,
+          15.7673228577021
+        ]
+      ]
+    ]
+  },
+  "links": [],
+  "assets": {
+    "xml": {
+      "href": "../tests/data-files/AST_L1T_00305032000040446_20150409135350_78838.hdf.xml",
+      "type": "application/xml",
+      "title": "XML metadata",
+      "roles": [
+        "metadata"
+      ]
+    },
+    "VNIR": {
+      "href": "../tests/data-files/external/AST_L1T_00305032000040446_20150409135350_78838-VNIR.tif",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "VNIR Swath data",
+      "eo:bands": [
+        {
+          "name": "VNIR_Band1",
+          "common_name": "green",
+          "description": "visible yellow/green",
+          "center_wavelength": 0.56,
+          "full_width_half_max": 0.08
+        },
+        {
+          "name": "VNIR_Band2",
+          "common_name": "red",
+          "description": "visible red",
+          "center_wavelength": 0.66,
+          "full_width_half_max": 0.06
+        },
+        {
+          "name": "VNIR_Band3N",
+          "common_name": "nir08",
+          "description": "near infrared",
+          "center_wavelength": 0.82,
+          "full_width_half_max": 0.08
+        }
+      ],
+      "proj:shape": [
+        4945,
+        5593
+      ],
+      "proj:bbox": [
+        251999.99997898546,
+        1670399.999999571,
+        335879.99999891786,
+        1744559.9999996407
+      ],
+      "proj:transform": [
+        14.997318079730448,
+        0.0,
+        251999.99997898546,
+        0.0,
+        -14.996966632976713,
+        1744559.9999996407,
+        0.0,
+        0.0,
+        1.0
+      ],
+      "raster:bands": [
```

Comment:
We've run into this before, but I can't remember the collection. The dictionaries in the `raster:bands` list have no identifying information to associate them with the `eo:bands` list elements. They are all the same, but I think it would be better if there was a `name` field in the `raster:bands` dictionaries.

Reply:
Yeah, I think that's a good idea, added.